src/app/user-page/user-page.component.ts

```diff
@@ -82,15 +81,20 @@ export class UserPageComponent extends Base implements OnInit {
     this.dialog.open(GithubAppsLogsComponent, { width: bootstrap4extraLargeModalSize, data: { userId: userId } });
   }
 
+  getOtherLinkedAccounts() {
+    this.usersService
+      .getUserTokens(this.user.id)
+      .pipe(takeUntil(this.ngUnsubscribe))
+      .subscribe((tokens: TokenUser[]) => {
+        for (const account of this.accountsInfo) {
+          const found = tokens.find((token) => token.tokenSource === account.source);
+          if (found && !this.publicAccountsSource.includes(account.source)) {
+            this.otherLinkedAccountsInfo.push(Object.assign({ username: found.username }, account));
```

Comment:
Might be a good idea to clear `otherLinkedAccountsInfo` before we start accumulating values to it, in case the component is recycled by Angular and there's already values in it.